packages/react-app/src/constants/Bankless.ts

```diff
@@ -4,6 +4,7 @@ import { CustomerProps } from '../models/Customer';
  */
 export const BANKLESS: CustomerProps = {
 	customerName: 'BanklessDAO',
+	customerKey: 'banklessdao',
```

Comment:
Is there a reason for a separate key rather than just the name (or derivation on the name e.g. <name>.toLowerCase)? Do we expect collisions on customer name?

Reply:
This is a good questions and will outline thoughts above to discuss